src/test/scala/uk/gov/nationalarchives/tdr/api/db/repository/ConsignmentRepositorySpec.scala

```diff
@@ -21,10 +21,23 @@ class ConsignmentRepositorySpec extends AnyFlatSpec with ScalaFutures with Match
 
     TestUtils.createConsignment(consignmentId, userId)
 
-    consignmentRepository.addParentFolder(consignmentId, Option("TEST PARENT FOLDER NAME")).futureValue
+    consignmentRepository.addParentFolder(consignmentId, Option("TEST ADD PARENT FOLDER NAME")).futureValue
 
     val parentFolderName = consignmentRepository.getConsignment(consignmentId).futureValue.map(consignment => consignment.parentfolder)
 
-    parentFolderName should contain only Some("TEST PARENT FOLDER NAME")
+    parentFolderName should contain only Some("TEST ADD PARENT FOLDER NAME")
+  }
+
+  "getParentFolder" should "get parent folder name for a consignment" in {
```

Comment:
It would also be good to have a test of the case where the name hasn't been saved yet, to show that the query doesn't throw an error.

Reply:
I'll add that in! :+1: